lib/curl_ntlm_core.c

```diff
@@ -52,8 +52,6 @@
 
 #if !defined(USE_WINDOWS_SSPI) || defined(USE_WIN32_CRYPTO)
 
-#ifdef USE_OPENSSL
```

Comment:
I would prefer if the inner USE_OPENSSL define was removed instead since that would keep the indentation. We can then re-indent lines 57-63 one step to the left  aligning them with the other SSL backends.

Reply:
We would also need to re-indent lines from 64 to 75. Is that ok?

Reply:
No those can stay where they are, as they are correctly indented two spaces in underneath the topmost `#ifdef USE_OPENSSL`

Reply:
Is that ok?